charts/brig/templates/deployment.yaml

```diff
@@ -42,50 +42,11 @@ spec:
         - name: "brig-secrets"
           secret:
             secretName: "brig"
-        {{- if .Values.config.geoip.enabled }}
-        - name: "geoip"
-          emptyDir: {}
-        {{- end }}
         {{- if eq (include "useCassandraTLS" .Values.config) "true" }}
         - name: "brig-cassandra"
           secret:
             secretName: {{ (include "tlsSecretRef" .Values.config | fromYaml).name }}
         {{- end}}
-      {{- if .Values.config.geoip.enabled }}
-      # Brig needs GeoIP database to be downloaded before it can start.
-      initContainers:
-        - name: geoipdownload
-          image: "{{ .Values.config.geoip.image.repository }}:{{ .Values.config.geoip.image.tag }}"
-          imagePullPolicy: {{ default "" .Values.config.geoip.imagePullPolicy | quote }}
-          volumeMounts:
-          - name: "geoip"
-            mountPath: "/usr/share/GeoIP"
-          # The environment variables are documented at:
-          # https://github.com/maxmind/geoipupdate/blob/62b34e648a842dc03ccf4ad3f61e2d85eaec98fc/doc/docker.md
-          env:
-          - name: GEOIPUPDATE_ACCOUNT_ID
-            valueFrom:
-              secretKeyRef:
-                name: brig-geoip
-                key: accountId
-          - name: GEOIPUPDATE_LICENSE_KEY
-            valueFrom:
-              secretKeyRef:
-                name: brig-geoip
-                key: licenseKey
-          - name: GEOIPUPDATE_EDITION_IDS
-            valueFrom:
-              secretKeyRef:
-                name: brig-geoip
-                key: editionIds
-          - name: GEOIPUPDATE_FREQUENCY
-            value: "0" # Setting this to 0 makes the script only run geoipupdate once.
-      {{- end }}
-        {{- if eq (include "includeSecurityContext" .) "true" }}
```

Comment:
weird that the second `if` is not included in the body of the first one, but removing it fixes that i guess...

Reply:
Was wondering about the same question. As far as I see, it would be inconsequent to leave this `if` remain, since it seems to be there for geoip only.